requirements.txt

```diff
@@ -4,3 +4,4 @@ pydocstyle==4.0.1
 pytest-flake8==1.0.4
 pytest-isort==0.3.1
 pytest==5.2.0
+pytest-mypy~=0.4.2
```

Comment:
Tools like this we might want to hard code to a specific version so that changes between version don't cause new errors.

Reply:
I've made it a fixed version, although I'm wondering whether semver should protect us from errors when automatically pulling patches.